yax/postgres/src/main/scala/doobie/postgres/free/pgconnection.scala

```diff
@@ -83,7 +85,7 @@ object pgconnection {
 
     // Primitive Operations
     case class  AddDataType(a: String, b: String) extends PGConnectionOp[Unit]
-    case class  AddDataType1(a: String, b: Class[_]) extends PGConnectionOp[Unit]
+    case class  AddDataType1(a: String, b: Class[_ <: PGobject]) extends PGConnectionOp[Unit]
```

Comment:
Why is this upper bound needed? Did the driver change the `.addDataType` method?